### DISCUSSION TITLE
Discussion answered badge missing?

### DISCUSSION THREAD

**Discussion by devkennyy** [community/community#28450] [Other Feature Feedback, Questions, & Ideas]
There used be a highlight below the profile that showed how many discussions you have answered, it's not longer there?

**Comment by jfullstackdev** [comment]
yup, it's missing for several days, maybe they will update that

**Reply by devkennyy** [reply]
I think it might be because the Galaxy Brain achievement was a little broken, they could be working on a fix

**Comment by thenithinbalaji** [comment]
I think they have removed that because Galaxy Brain is already an indicator of the number of discussions answered.
